docs/docs/managing-suggestions.md

```diff
@@ -13,45 +13,51 @@ _**Key:**_
 
 ## DM Responses
-DM responses are sent to the suggestion submitter based on various actions. DM responses may be toggled via `config dmResponses <true/false>`. This means DMs to the suggestion sumbitter will or won't not be sent via these actions based on how you configure the bot for your server:
+DM responses are sent to the suggestion submitter based on various actions. DM responses may be toggled by running `config dmResponses <true/false>`. This means DMs to the suggestion sumbitter will or won't be sent via these actions based on how you configure the bot for your server:
 
-- Suggestion submitted
-- Suggestion approved
-- Suggestion rejected
-- Suggestion note added
+- suggestion submitted
+- suggestion approved
+- suggestion rejected
+- suggestion note added
 
-By default, DM responses are enabled. This can be configured by a user with the Manager Server (`MANAGE_GUILD`) permission.
+By default, DM responses are enabled. This can be configured by an administrator.
 
 ## Setting Up Staff Roles
 
-If you haven't already done so, please refer to the [Getting Started](README.md#set-up-the-bot) section to set up staff roles. By default, users with the `MANAGE_GUILD` permission can manage suggestions. However, if you want staff members to have access to these commands, please refer to the aforementioned section.
+If you haven't already done so, please refer to the [Getting Started](README.md#set-up-the-bot) section to set up staff roles. By default, only administrators can manage suggestions. However, if you want other staff members to have access to these commands, please refer to the aforementioned section.
 
 ## Approving Suggestions
 
-Suggestions are approved via the sID. Once you have it, run the command `approve <ID>`. Afterward, series of actions will happen:
+Suggestions are approved via the sID. Once you have it, run the command `approve <sID|message ID>`. Afterward, series of actions will happen:
 
-* The suggestion will update as "Approved" in the suggestions channel and shortly delete itself afterward
-* The user will receive a DM that you approved their suggestion
-* The results of that suggestion will be logged in the suggestions logs channel
-* _An optional response can be added doing_ `approve <ID> [response]`
+* The suggestion will update as "Approved" in the suggestions channel and shortly delete itself afterward.
+* The user will receive a DM that you approved their suggestion.
+* The results of that suggestion will be logged in the suggestions logs channel.
+* _An optional response can be added by running_ `approve <sID|message ID> [response]`.
 
 ![Suggestion Approved Status](/images/managing-suggestions-3.png)
 
 ![Suggestion Approved DM](/images/managing-suggestions-4.png)
 
-It seems like a lot doesn't it? It really isn't because it all happens in the matter of seconds!
```

Comment:
Honestly, it can probably be said as "milliseconds" instead of "seconds." If an act like this is taking a couple of seconds, I'd be concerned.